src/Network/Network/Az.Network.psd1

```diff
@@ -436,6 +436,8 @@ CmdletsToExport = 'Add-AzApplicationGatewayAuthenticationCertificate',
                'Add-AzVirtualHubRoute', 'New-AzVirtualHubRouteTable', 
                'Add-AzVirtualHubRouteTable', 'Get-AzVirtualHubRouteTable', 
                'Remove-AzVirtualHubRouteTable', 'New-AzRoutingPolicy', 
+               'Get-AzVHubEffectiveRoute', 'Get-AzVHubInboundRoute',
```

Comment:
@zaynchen115 
Refer to our [naming best practice](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/cmdlet-best-practices.md#specific-noun-and-noun-singularity), you should use singular name for Get-AzVHubEffectiveRoutes, Get-AzVHubInboundRoutes and Get-AzVHubOutboundRoutes. As you make breaking change to a new added cmdlet `GetAzureRmVirtualHubEffectiveRoutesCommand`, if we don't include your PR into this release, the new added cmdlet will be not allowed to change in a breaking way until next major release. And so we change your cmdlet names from plural to singular and include them into this release.
Please follow the code-freeze date for the next time.